test-network-function/common/env.go

```diff
@@ -91,47 +75,23 @@ func GetOcDebugImageID() string {
 	return os.Getenv("TNF_OC_DEBUG_IMAGE_ID")
 }
 
-// logLevel retrieves the LOG_LEVEL environement vaiable
+// logLevel retrieves the LOG_LEVEL environment variable
 func logLevel() string {
-	return os.Getenv("LOG_LEVEL")
-}
-
-// stringToLogLevel converts a string to a log logrus level
-func stringToLogLevel(logLevelString string) (logrus.Level, error) {
-	logLevelString = strings.ToLower(logLevelString)
-	switch logLevelString {
-	case logLevelTraceString:
-		return logrus.TraceLevel, nil
-	case logLevelDebugString:
-		return logrus.DebugLevel, nil
-	case logLevelInfoString:
-		return logrus.InfoLevel, nil
-	case logLevelWarnString:
-		return logrus.WarnLevel, nil
-	case logLevelErrorString:
-		return logrus.ErrorLevel, nil
-	case logLevelFatalString:
-		return logrus.FatalLevel, nil
-	case logLevelPanicString:
-		return logrus.PanicLevel, nil
-	case logLevelEmptyString:
-		return logLevelDefault, errors.New(errorEmpty)
+	logLevel := os.Getenv("LOG_LEVEL")
+	if logLevel == "" {
+		log.Info("LOG_LEVEL environment is not set, defaulting to DEBUG")
+		logLevel = "debug" //nolint:goconst
```

Comment:
Do we really want the log level to be "debug" in case the environment variable is not set? The README.md file should be updated accordingly regarding this env var.